vispy/scene/widgets/widget.py

```diff
@@ -130,6 +134,54 @@ def inner_rect(self):
         return Rect((m, m), (self.size[0]-2*m, self.size[1]-2*m))
 
     @property
+    def stretch(self):
+        """Stretch factors (w, h) used when determining how much space to
+        allocate to this widget in a layout.
+        
+        If either stretch factor is None, then it will be assigned when the
+        widget is added to a layout based on the number of columns or rows it
+        occupies.
+        """
+        return self._stretch
+
+    @stretch.setter
+    def stretch(self, s):
+        self._stretch = s
+        self._update_layout()
+
+    @property
+    def fixed_size(self):
+        """Fixed size (w, h) of the widget.
+        
+        Specifying a fixed size for either axis forces the widget to have a 
+        specific size in a layout. Setting either axis to None allows the 
+        widget to be resized by the layout.
```

Comment:
There is a contradiction here. What happens if I provide `None, 100`? As I understand it, this allows a widget to override a layout? Will the layout simply ignore the widget?